### PR TITLE
removed roles and apply button, added cow.

### PR DESCRIPTION
**Pull request by eliotsaylor**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html><html><head>
-    <title>HackDavis Recruitment</title>
+    <title>HackDavis | January 2019</title>
```
Comment (klanmiko) [inline index.html:2]: Please see the updated README on how to deploy this website. Make these changes on recruitment.html and then use the scripts to render the website onto this, or you could just copy over these changes to that file manually.

File: scripts/initial_map.json [hunk 2]
```diff
@@ -1,115 +1,82 @@
 {
```
Comment (klanmiko) [inline scripts/initial_map.json]: This file does not have to be changed, I think you ran scripts/compile_current.js

File: package.json [hunk 3]
```diff
@@ -9,6 +9,8 @@
     "jquery": "^3.3.1",
     "jsdom": "^11.5.1",
     "node-sass": "^4.7.2",
+    "nodemon": "^1.17.3",
+    "react": "^16.3.2",
```
Comment (klanmiko) [inline package.json]: Also why have these packages been added?

**Review by klanmiko — CHANGES_REQUESTED** [review]
See build system in README for more details. Otherwise the website itself looks good

**Review by klanmiko — CHANGES_REQUESTED** [review]
Still need to modify recruitment.html manually to update the template

**Comment by jryio** [comment]
Moo

**Review by klanmiko — APPROVED** [review]

**Pull request merged**